walkthroughs/nbconverted/single_cell_usage.py

```diff
@@ -0,0 +1,248 @@
+annotate(
+    profiles=sc_profile,
+    platemap=platemap_df,
+    join_on=["Metadata_well_position", "Image_Metadata_Well"],
+    output_file=anno_profiles_path,
+    compression_options="gzip",
+)
+
+# save message display
+print(f"Annotated profile saved in: {anno_profiles_path}")
+
+
+# ## Normalization Step
+# 
+# The next step is to normalize our dataset using the `normalize` function provided by `pycytominer`.
+# More information regards `pycytominer`'s `normalize` function can be found [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.normalize)
+# 
+# Normalization is a critical preprocessing step that improves the quality of our dataset. It addresses two key challenges: mitigating the impact of outliers and handling variations in value scales. By normalizing the data, we ensure that our downstream analysis is not heavily influenced by these factors.
+# 
+# Additionally, normalization plays a crucial role in determining feature importance (which is crucial for our last step). By bringing all features to a comparable scale, it enables the identification of important features without biases caused by outliers or widely-scaled values.
+# 
+# To normalize our annotated single-cell morphology profile, we will utilize the `normalize` function from `pycytominer`. This function is specifically designed to handle the normalization process for cytometry data. 
+
+# In[7]:
+
+
+# load annotated profile
+annotated_df = load_profiles(anno_profiles_path)
+
+# normalize dataset
+normalize(
+    profiles=annotated_df,
+    features="infer",
+    image_features=False,
+    meta_features="infer",
+    samples="all",
+    method="standardize",
+    output_file=norm_profiles_path,
+    compression_options="gzip",
+)
+
+# save message display
+print(f"Normalized profile saved in: {anno_profiles_path}")
+
+
+# ## Feature Selection
+# 
+# 
+# In the final section of our walkthrough, we will utilize the normalized dataset to extract important morphological features and generate a selected features profile.
+# 
+# To accomplish this, we will make use of the `feature_select` function provided by `pycytominer`. 
+# Using `pycytominer`'s `feature_select` function to our dataset, we can identify the most informative morphological features that contribute significantly to the variations observed in our data. These selected features will be utilized to create our feature profile.
+# 
+# For more detailed information about the `feature_select` function, its parameters, and its capabilities, please refer to the documentation available [here](https://pycytominer.readthedocs.io/en/latest/pycytominer.html#module-pycytominer.feature_select).
+# 
+
+# In[8]:
+
+
+# loading normalized profile
+normalized_df = load_profiles(norm_profiles_path)
+
+# creating selected features profile
+feature_select(
+    profiles=normalized_df,
+    features="infer",
+    image_features=False,
+    samples="all",
+    operation="variance_threshold",
```

Comment:
Can you apply more than one? It would be good for a user to see how to apply multiple feature selection operations.

See @jenna-tomkinson's notebook in cell 3 https://github.com/WayScience/nf1_cellpainting_data/blob/main/3.processing_features/3.feature_select_sc.ipynb